res/layout/instant_onboarding_activity.xml

```diff
@@ -0,0 +1,131 @@
+                 android:transitionName="avatar"/>
+
+      <com.google.android.material.textfield.TextInputLayout
+          android:id="@+id/name"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginLeft="10dp">
+
+        <org.thoughtcrime.securesms.components.emoji.EmojiEditText
+            android:id="@+id/name_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/pref_your_name"
+            android:inputType="textCapWords" />
+      </com.google.android.material.textfield.TextInputLayout>
+
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/information_label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="16dp"
+        android:gravity="start"
+        android:text="@string/set_name_and_avatar_explain"
+        android:textColor="@color/gray50" />
+
+    <TextView
+        android:id="@+id/privacy_policy_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="start"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="16dp"
+        android:text="@string/instant_onboarding_agree_default"
+        android:textColor="@color/delta_accent"
+        android:textSize="16sp"/>
+
+    <Button
+        style="@style/ButtonPrimary"
+        android:id="@+id/signup_button"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingLeft="16dp"
+        android:paddingRight="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginBottom="16dp"
+        android:text="@string/instant_onboarding_create"/>
+
+    <androidx.legacy.widget.Space
+        android:layout_weight="2"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"/>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+      <TextView
+          android:id="@+id/other_options_button"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintEnd_toStartOf="@+id/separator"
+          app:layout_constraintTop_toTopOf="parent"
+          app:layout_constraintBottom_toBottomOf="parent"
+          android:gravity="start"
+          android:padding="16dp"
+          android:text="@string/instant_onboarding_show_more_instances"
+          android:textColor="@color/delta_accent"
+          android:textSize="16sp"/>
```

Comment:
i suggest to tune the buttons a bit own, using the color&font of the subtitle, this is esp. nicer with the keyboard visible as well and mimics more what desktop is doing (i know this is not visible as a button at the first glance then, but that is how things are nowadays - and in this case it is okay; the more the other things stand out - and by its position it is also clear it is tappable)

```suggestion
          android:textColor="@color/gray50"/>
```

<img width=220  src=https://github.com/deltachat/deltachat-android/assets/9800740/5e8c2dd0-7099-4a07-beb0-ed0bb1064455>